### DISCUSSION TITLE
为什么我的设置里没有“立即检查所有更新”这个选项呢？

### DISCUSSION THREAD

**Discussion by a86582751** [the1812/Bilibili-Evolved#3094] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/81522189/157170977-2175a11b-770b-46cd-a2d5-f4c61f1e24ea.png)
好奇怪。看更新日志里作者还特意提了这个功能

**Comment by the1812** [comment, marked as answer]
在预览版里的自动更新器里
